changelog/139.feature.rst

```diff
@@ -0,0 +1 @@
+A series of functions were added from `sunkit_image.asda` by declassing the code.
```

Comment:
This changelog needs to explain how to transition from the class to using the functions if you were already using asda.

Reply:
on it 👍

Reply:
And should I also mention the change in parameters in all those functions?